codespell_lib/data/dictionary.txt

```diff
@@ -7074,6 +7075,9 @@ corresponging->corresponding
 corresponing->corresponding
 corresponsding->corresponding
 corresponsing->corresponding
+correspont->correspond
```

Comment:
Can you add the correspondent and correspondence versions of these typos too (and the plural versions of both).